appserver/packager/appserver-base/src/main/docs/README.txt

```diff
@@ -1,4 +1,4 @@
-Thank you for downloading Payara Server 5.193!
+Thank you for downloading Payara Server 5.194!
```

Comment:
5.201-snapshot

Reply:
I believe 5.194 is correct, people are downloading the release source files not master branch